modules/gltf/gltf_document.cpp

```diff
@@ -30,6 +30,7 @@
 
 #include "gltf_document.h"
 
+#include "core/object/class_db.h"
```

Comment:
As you can see here the includes are grouped with `gltf_` local includes first, then general core/scene/editor/etc. headers next (absolute paths). So this should go in the next group.